Reviews/337 -  Royal web wallet and casper connector/1- Development/README.md

```diff
@@ -0,0 +1,394 @@
+
+* Set the ETH network to Rinkeby
+![metamask testnet](assets/metamask_testnet_1.png "metamask_1")
+
+* Connect wallet using metamask
+![metamask testnet](assets/metamask_testnet_2.png "metamask_2")
+
+![metamask testnet](assets/metamask_testnet_3.png "metamask_2")
+
+The informations about the current connected test ETH wallet and  the available test ETH balance seem to be wrong.
+The reviewer thinks that connecting to ETH networks via metamask  does not work properly in the application.
+But he cannot confirm that since there are no errors indicating it on the web console.
+The page shows an error message (Can't load ethereum contract. Please check configuration).This error is most likely due to the missing variables we could not set above.
+
+**The OP is strongly recommended to give a more detailled step-by-step guide on how to use the application**
+
+2- Connect Casper Signer
+
+The reviewer was able to connect to the Casper testnet via CasperSigner. The public key of Caper signer wallet was shown on the page. 
+**The balance of the connected account (1000 test cspr) was not displayed on the page 
+and an error keeps  being thrown in the web console (Chrome).**
+
+```console
+Access to fetch at 'http://138.201.54.44:7777/rpc' from origin 'http://localhost:3000' 
+has been blocked by CORS policy: Response to preflight request doesn't pass access control 
+check: No 'Access-Control-Allow-Origin' header is present on the requested resource. 
+If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the 
+resource with CORS disabled.
+```
+
+
+![Casper testnet](assets/casper_testnet.png "metamask")
+
+3. Grab ETH Faucet (for testnet) 
+
+The reviewer could not use his own testnet ETH wallet  on the Rinkeby ETH test network.
+  
+4. Grab CSPR Faucet (for testent)
+   
+The reviewer used Casper Testnet Faucet to claim 1000 test cspr.
+
+5. Enter amount = 1
+
+6. Push Exchange
+
+Hitting the Exchange button throws an error on the page : **ethContractError**
+
+
+
+## Overall Impression of usage testing
+
+The reviewer was able to build and run the application. 
+**However, the build instructions are not very clear and some important informations are missing, especially on the use of configuration variables and how to set them in the project.**
+**It is also not possible to successfully run the whole test scenario provided in the README file.
+The reviewer managed to connect his metamask and  casper signer  wallets, but the ETH address displayed and the balance of the ETH wallet do not seem to match.
+Also , the web console shows continuous errors after connecting to the casper signer wallet**
+
+
+
+Requirement | Finding
+------------ | -------------
+Project builds without errors | PASS with notes
+Documentation provides sufficient installation/execution instructions | PASS with notes
+Project functionality meets/exceeds acceptance criteria and operates without error | Failed
```

Comment:
As per the CRDAO template for the DxD submissions, it needs to be `PASS`, `FAIL` or `PASS with Notes`. Attention to detail is appreciated.

Reply:
done , will pay attention next time

Reply:
I'm afraid it's improved but not done. `Fail` and `FAIL` are not the same. We may need to parse these values in the future.

Reply:
Yes, sorry, done !